libs/util/code-gen-utils/README.md

```diff
@@ -0,0 +1,11 @@
+# util-code-gen-utils
+
+This library was generated with [Nx](https://nx.dev).
+
+## Building
+
+Run `nx build util-code-gen-utils` to build the library.
+
+## Running unit tests
+
+Run `nx test util-code-gen-utils` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
Please add a separate task to update the README file

Reply:
https://github.com/amplication/amplication/issues/4697